lighthouse-core/report/v2/renderer/category-renderer.js

```diff
@@ -484,6 +529,12 @@ class CategoryRenderer {
     const passedElem = this._renderPassedAuditsSection(passedElements);
     element.appendChild(passedElem);
 
+    // don't create a not applicable section if there are no not applicables
+    if (!notApplicableElements.length) return element;
```

Comment:
seems like we need to move these `if (!array.length)` checks inside some braces instead of early return

might be a current bug with no manual audits if everything fails :)